roles/installer/templates/deployments/deployment.yaml.j2

```diff
@@ -46,14 +46,16 @@ spec:
       priorityClassName: '{{ control_plane_priority_class }}'
 {% endif %}
       initContainers:
-{% if bundle_ca_crt or projects_persistence|bool or init_container_extra_commands %}
         - name: init
           image: '{{ _init_container_image }}'
           imagePullPolicy: '{{ image_pull_policy }}'
           command:
             - /bin/sh
             - -c
             - |
+              hostname=$MY_POD_NAME
+              receptor --cert-makereq bits=2048 commonname=$hostname dnsname=$hostname nodeid=$hostname outreq=/etc/receptor/tls/receptor.req outkey=/etc/receptor/tls/receptor.key
+              receptor --cert-signreq req=/etc/receptor/tls/receptor.req cacert=/etc/receptor/tls/ca/receptor-ca.crt cakey=/etc/receptor/tls/ca/receptor-ca.key outcert=/etc/receptor/tls/receptor.crt verify=yes
```

Comment:
What does receptor on the TLS client cert/key that openssl (or the community.crypto collection) coudn't do ?
I mean if it's only for creating and signing TLS certificates then we don't really need to have two different workflows for the CA and client certificates.
Also, any reason why the TLS client certificate and key aren't stored as a k8s Secret like for the CA ? This would mean that the TLS client certificate/key will be recreate each time the deployment recreate (like during upgrades) but I don't know if that's an issue or not.

Reply:
it is intended for the client cert/key to be regenerated for each new instance so that the instance will have a unique identity

Reply:
this runs in the init container that's why we did not use ansible playbook 

using receptor to generate it is just for connivance and yes receptor does put 

# Red Hat has an OID namespace (RHANANA). Receptor has its own designation under that.
RECEPTOR_OID = "1.3.6.1.4.1.2312.19.1"

in the SAN